apm-agent-plugins/apm-dubbo-plugin/pom.xml

```diff
@@ -39,7 +39,7 @@
         <dependency>
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
-            <version>3.14.7</version>
+            <version>4.12.0</version>
```

Comment:
Apparently doesn't hurt to use the latest version here, so let's stick with it I'd say